cluster-cm/src/main/java/com/sequenceiq/cloudbreak/cm/ClouderaManagerModificationService.java

```diff
@@ -203,6 +204,35 @@ public void upgradeClusterRuntime(Set<ClusterComponent> components) throws Cloud
         }
     }
 
+    private void restartServices(ClustersResourceApi clustersResourceApi) throws ApiException, CloudbreakException {
+        List<String> serviceNames = getClusterServices();
+        ApiRestartClusterArgs restartClusterArgs = new ApiRestartClusterArgs();
+        restartClusterArgs.setRestartServiceNames(serviceNames);
```

Comment:
I think this setting is not needed, as per API docs: "Only restart services that are specified and their dependent services". Omitting it should work